commands/root.go

```diff
@@ -55,6 +55,7 @@ the same git remote you are already using to collaborate with other people.
 	}
 
 	env := execenv.NewEnv()
+	cmd.PersistentFlags().StringArrayVarP(&env.RepoPath, "repo-path", "C", []string{}, `Run as if git-bug was started in <path> instead of the current working directory.`)
```

Comment:
as discussed in the comment thread, i think this should probably be a `string` (that is, not a slice of strings).